src/App.tsx

```diff
@@ -1,56 +1,86 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import './App.scss';
+import debounce from 'lodash.debounce';
 import { peopleFromServer } from './data/people';
+import { SearchBar } from './component/SearchBar';
+
+const preparedPeople = [...peopleFromServer];
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [people] = useState(preparedPeople);
+  const [filteredPeople, setFilteredPeople] = useState(people);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [focused, setFocused] = useState(false);
 
-  return (
-    <main className="section">
-      <h1 className="title">
-        {`${name} (${born} = ${died})`}
-      </h1>
+  const inputField = useRef<HTMLInputElement>(null);
 
-      <div className="dropdown is-active">
-        <div className="dropdown-trigger">
-          <input
-            type="text"
-            placeholder="Enter a part of the name"
-            className="input"
-          />
-        </div>
+  const { name, born, died } = filteredPeople?.length
+    ? filteredPeople[0] : { name: '', born: '', died: '' };
```

Comment:
This is wrong - you should be able to select the person - not just pick one from the top